app/models/driver.rb

```diff
@@ -0,0 +1,43 @@
+class Driver < ApplicationRecord
+  has_many :trips, dependent: :nullify
+
+
+  def all_trips
+    temp_trips = []
+    all_trips = []
+    temp_trips << Trip.where(driver_id: id)
+    temp_trips.each do |trips|
+      trips.each do |trip|
+        all_trips << trip
+      end
+    end
+    return all_trips
+  end
```

Comment:
This method returns an array of all trips associated with this `Driver`.

Why couldn't we just use the built in `.trips` method? The built in `.trips` method returns an array of all trips associated with this instance of `Driver`.